docs/guides/security-policy-for-authors.md

```diff
@@ -131,20 +154,19 @@ For example,
 
 or [\[1\]](#references-and-notes) (Github-Sec-Advisory)
 
-> Security vulnerabilities can be reported via our GitHub repository
-> at https://example.github.com/foobar.  On the "Advisories" page you
-> can click on the "Report a vulnerability" button.
+> Security vulnerabilities can be reported via the project GitHub repository
+> [Security Advisories](https://example.github.com/foobar/security/advisories).
+> On the "Advisories" page you can click on the "Report a vulnerability" button.
 
-or
+or for GitLab
 
-> Security vulnerabilities can be reported via our GitLab repository a
-> https://gitlab.com/foobarkeepers/foobar/issues.  When reporting the
-> issue, please check the tickbox on the form labelled "This issue is
-> confidential and should only be visible to team members".
+> Security vulnerabilities can be reported via our GitLab repository
+> [issues](https://gitlab.com/foobarkeepers/foobar/issues).  When
+> reporting the issue, please check the tickbox on the form labelled
+> "This issue is confidential and should only be visible to team members".
```

Comment:
this stuff is subject to change,  Maybe more generic?

Reply:
Can you elaborate?

Maybe we should note that there is something similar for GitLab.